.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '[0-9]+\.[0-9]+\.[0-9]+'
+  pull_request:
+    branches:
+      - master
+
+env:
+  # https://github.com/actions/virtual-environments/tree/main/images/macos
+  DEVELOPER_DIR: /Applications/Xcode_12.4.app/Contents/Developer
+
+  WORKSPACE: Alicerce.xcworkspace
+  SCHEME: Alicerce
+
+  IOS_SDK: iphonesimulator
+  IOS_DESTINATION: "platform=iOS Simulator,name=iPhone 12 Pro,OS=latest"
+
+  DERIVED_DATA_PATH: build
+
+jobs:
+  env-details:
+    name: Environment details
+    runs-on: macOS-latest
```

Comment:
That's a good point 🤔 

Perhaps we should pin it to a specific image (with macOS and Xcode version) to prevent surprises in the future.
Oh the other hand, having it break due to version upgrades once in a while gives us a nudge to fix stuff and migrate to the latest versions 😆 

Decisions... decision 🙈

Reply:
It appears that counter-intuitively, the `macOS-latest` is actually `macOS-10.15` (macOS Catalina). 

Source: [GitHub Actions supported runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources)